sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/changelog.md

```diff
@@ -3,6 +3,15 @@
 ## Current version
 ###  Feature Additions
 
+## Version 4.1.1
+###  Feature Additions

```

Comment:
Copy this whole section to csproj file for Datafactory, and update the version there to 4.1.1.
Also update AssemblyFileVersion to 4.1.1.0 in Properties

Reply:
I guess you meant changing both the AssemblyFileVersion and the AssemblyVersion under the AssemblyInfo.cs file. I changed them both to 4.1.1.0.